### DISCUSSION TITLE
Vult modules?

### DISCUSSION THREAD

**Discussion by mervenca** [DISTRHO/Cardinal#460] [General]
Sorry if already discussed or explaines somewhere, but would it be possible to have vult modules (trummors and lapsus)  in cardinal aswell, or are they somehow restricted for doing that?

Thanks! And delete my thread if its already asked an answered somewhere :)

**Comment by falkTX** [comment]
Vult modules are not opensource.
They kinda pretend they are with a github repo https://github.com/modlfo/VultModules but there is no public source code for their modules.

Nothing we can do here.

**Comment by dromer** [comment]
This has been asked several times already. Please think about such questions and find out if there is source-code for modules.

If there is not then obviously they cannot be added to the project and you don't have to make a request for them.

**Comment by falkTX** [comment]
I think the question is fair. the https://github.com/modlfo/VultModules page really leads the impression of it being open project on github, similar to nysthi. but both are proprietary.
likely again due to VCV's awful policy on not allowing commercial opensource plugins, devs that want to have commercial options cannot have their free versions open, really sucks
